changelog.txt

```diff
@@ -1,5 +1,7 @@
 == Changelog ==
 
+* Tweak - Added $ticket_id to 'tribe_events_tickets_metabox_edit_ajax_advanced'. [ETP-111]
```

Comment:
```suggestion
= [5.1.6] TBD =

* Tweak - Added `$ticket_id` parameter to the `tribe_events_tickets_metabox_edit_ajax_advanced` filter. [ETP-111]
```
Seems like the `changelog.txt` is out of synch, good opportunity to set the upcoming version there as well. Not part of the fix, but I normally prefer to fix them as soon as they see them so we can improve things as we get going.